wdl-lint/tests/lints/description-missing/source.errors

```diff
@@ -14,6 +14,17 @@ note[DescriptionMissing]: workflow `bar` is missing a description key
    │
    = fix: add a `description` key to the meta section
 
+note[SectionOrdering]: sections are not in order for struct `Baz`
```

Comment:
Can you edit `source.wdl` so that the order is correct and this diagnostic goes away?